corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -868,6 +932,11 @@ def _number_columns(self):
         return [col["field"] for col in self._report_columns if col.get("aggregation", None) in ["avg", "sum"]]
 
     @property
+    @memoized
```

Comment:
No need to cache a boolean constant. You could still memoize in subclasses if the value needs to be calculated for them.